js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,31 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var ClubhouseOpenButton = new Lang.Class({
+    Name: 'ClubhouseOpenButton',
+    Extends: St.Button,
+
+    _init: function(params) {
+        this.parent(params);
+
+        this._icon = new St.Icon({ style_class: 'clubhouse-open-button-icon' });
+        this.child = this._icon;
+    },
+
+    setHighlighted: function(highlighted=true) {
```

Comment:
Don't use a default value here. I think it's better to explicitely see `setHighlighted(true)` than an empty method call.

Reply:
I think this is a matter of taste. But I will do as you say to move this PR forward.

Reply:
I haven't seen this used before in the Shell so I thought it'd be better not to use it (besides the logic about `setHighlighted` hinting it's going to set something), but I may be mistaken.

Reply:
I did grep the codebase prior to use it and I've found a few cases. But now I see they are more the exception than the rule.